### DISCUSSION TITLE
Config is ignored

### DISCUSSION THREAD

**Discussion by serhez** [rose-pine/neovim#93] [Q&A]
Hello, I have a setup such as:

```lua
require("rose-pine").setup({
    --- @usage 'main' | 'moon'
    dark_variant = "moon",
    bold_vert_split = false,
    dim_nc_background = false,
    disable_background = false,
    disable_float_background = false,
    disable_italics = false,
})
```

However, the `main` dark variant is the one being applied, regardless of the fact I'm specifying the `moon` variant on the config.

Furthermore, I tried disabling italics with `disable_italics = false` but this also does not work, which makes me think none of the config is being loaded.

If it helps, you can find my whole config [here](https://github.com/serhez/nvim-config); you may want to look at `lua/plugins/init.lua` for packer config and the list of plugins, and `lua/plugins/configs/colorschemes/rose-pine/init.lua` for this plugin's config.

Any ideas?

**Comment by mvllow** [comment]
Make sure you are setting your colorscheme after setting the config. If you are able to reproduce with a minimal config please feel free to open an issue :)

**Reply by tinnick** [reply]
Same issue here. The example below does not work for me:
```lua
vim.cmd [[packadd packer.nvim]];

return require('packer').startup(function(use)
  use({
    'rose-pine/neovim',
    as = 'rose-pine',
    config = function()
      require('rose-pine').setup({ dark_variant = 'moon' });
      vim.cmd('colorscheme rose-pine');
    end
  });
end);
```

nvim version is 0.9.0

**Reply by tinnick** [reply]
I tried moving it to a different file (and confirm that it is imported) with the exact same impl. as the README and can confirm that it still doesn't work.

**Reply by mvllow** [reply]
Using a fresh install of Neovim `NVIM v0.9.0-dev-1742+g45d1b1c67-dirty` and clearing out `~/.cache/nvim` and `~/.local/share/nvim` with the above config in `~/.config/nvim/init.lua` works for me. Unfortunately I can't reproduce any issues mentioned here :(

**Comment by Pkcarreno** [comment]
I have a similar problem, in my case I am loading the config from the `after` folder and my file is called `rose-pine.rc.lua` and for some reason it does not read it, I think the problem is in the file name and the module name, I tried to rename the file to `neovim.rc.lua` but it did not catch it either, any idea?

**Reply by mvllow** [reply]
Try using `after/plugin/rose-pine.rc.lua`. This worked for me, using the below packer config:

```lua
require('packer').startup(function(use)
	use({ 'rose-pine/neovim', as = 'rose-pine' });
end);
```

**Reply by Pkcarreno** [reply]
thank you very much for answering, I had partially solved this problem, it seems that there was some incompatibility between this library and colorbuddy which I used together in the same file `rose-pin.rc.lua`, that's why the whole file  was not executed and I couldn't mount the color scheme with the command, what I did was to remove colorbuddy and it worked fine.
